.circleci/config.yml

```diff
@@ -508,101 +518,54 @@ parameters:
     type: boolean
     default: false
 
-  target-env:
-    type: string
-    default: ''
-  owasp:
-    type: boolean
-    default: false
-
-  qasp:
+  run_owasp_scan:
     type: boolean
     default: false
 
-  pa11y:
-    type: boolean
-    default: false
-
-  frontend:
-    type: boolean
-    default: false
-
-  backend:
-    type: boolean
-    default: false
+  target_env:
+    type: string
+    default: ''
 
 workflows:
-  nightly:
-    jobs:
-      - nightly-owasp-scan
-    triggers:
-      - schedule:
-          cron: "0 0 * * *"
-          filters:
-            branches:
-              only:
-               - raft-tdp-main
-
   build-and-test:
-
-    unless: << pipeline.parameters.run_dev_deployment >> 
-  # Remove this check in a follow up PR to the work for 1013
+    unless:
+      or:
+        - << pipeline.parameters.run_dev_deployment >>
+        - << pipeline.parameters.run_owasp_scan >>
     jobs:
       - secrets-check
       - test-frontend:
-          owasp: << pipeline.parameters.owasp >>
-          pa11y: << pipeline.parameters.pa11y >>
-          requires: 
+          requires:
             - secrets-check
-
       - test-backend:
-          owasp: << pipeline.parameters.owasp >>
-          requires: 
+          requires:
             - secrets-check
 
-  build-and-test-backend:
-    unless:
-      or:
-        - << pipeline.parameters.run_dev_deployment >> 
-        - not: << pipeline.parameters.backend >> 
-        - << pipeline.parameters.qasp >> 
+  dev-deployment:
+    when: << pipeline.parameters.run_dev_deployment >>
     jobs:
-      - secrets-check
-      - test-backend:
-          owasp: << pipeline.parameters.owasp >>
-          requires: 
-            - secrets-check
+      - deploy-infrastructure-dev
+      - deploy-dev:
+          target_env: << pipeline.parameters.target_env >>
+          requires:
+            - deploy-infrastructure-dev
 
-  build-and-test-frontend:
-    unless:
-      or:
-        - << pipeline.parameters.run_dev_deployment >> 
-        - not: << pipeline.parameters.frontend >> 
-        - << pipeline.parameters.qasp >> 
+  nightly:
     jobs:
-      - secrets-check
-      - test-frontend:
-          owasp: << pipeline.parameters.owasp >>
-          pa11y: << pipeline.parameters.pa11y >>
-          requires: 
-            - secrets-check
+      - nightly-owasp-scan
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
+          filters:
+            branches:
+              only:
+               - raft-tdp-main
 
-  build-and-test-qasp:
-    unless:
-      or:
-        - << pipeline.parameters.run_dev_deployment >> 
-        - not: << pipeline.parameters.qasp >> 
+  owasp-scan:
```

Comment:
This is the new workflow, that only runs the OWASP scans and is triggered  by passing in the `run_owasp_scan` pipeline parameter. This gets accomplished through use of the GitHub Action that runs when the `QASP Review` label is present on a PR.